pydevices/acq400Devices/acq400_base.py

```diff
@@ -92,7 +88,51 @@ def init(self):
         if trg == 'automatic':
             uut.s0.transient = 'SOFT_TRIGGER=1'
 
-        uut.s0.transient = "POST={}".format(self.samples.data())
+        # If PRE samples different from zero
+        presamples = 100000
+        # presamples = 0
+        uut.s0.transient = "PRE={} POST={}".format(presamples, self.samples.data())
+
+        #Setting WR Clock to 20MHz by first being sure that MBCLK FIN is in fact = 0
+        uut.s0.SIG_CLK_MB_FIN = '0'
+
+        #Set the Sampling rate in the ACQ:
+        # MB Clock (WR Clock): 20MHz
+        # mb_freq = uut.s0.SIG_CLK_MB_FREQ
+        mb_freq = 20000000.00 #MHz
+
+        self.dprint(1, "Setting sample rate to %r Hz", self.freq.data())
+        clockdiv      = mb_freq/self.freq.data()
+
+        uut.s1.CLKDIV = "{}".format(clockdiv)
+
+        acq_sample_freq = uut.s0.SIG_CLK_S1_FREQ
+        
+        self.dprint(1, "After setting the sample rate the value in the ACQ is%r Hz", acq_sample_freq)
+
+        #The following is what the ACQ script called "/mnt/local/set_clk_WR" does to set the WR clock to 20MHz
+        uut.s0.SYS_CLK_FPMUX     = 'ZCLK'
+        uut.s0.SIG_ZCLK_SRC      = 'WR31M25'
+        uut.s0.set_si5326_bypass = 'si5326_31M25-20M.txt'
```

Comment:
What is this? A text file?

Reply:
i think its a defined file on the dtacq device. that hopefully comes with the firmware

Reply:
Yes, exactly. It's in here: 
/etc/si5326.d/si5326_31M25-20M.txt